java/src/main/java/com/logicalclocks/hsfs/FeatureGroup.java

```diff
@@ -115,80 +115,175 @@ public FeatureGroup(FeatureStore featureStore, int id) {
     this.id = id;
   }
 
+  /**
+   * Read the feature group into a dataframe.
+   * Reads the feature group by from the offline storage as Spark DataFrame on Hopsworks and Databricks.
+   *
+   * <pre>
+   * {@code
+   *        // get feature store handle
+   *        FeatureStore fs = ...;
+   *        // get feature group handle
+   *        FeatureGroup fg = ...;
+   *        // read feature group
+   *        fg.read()
```

Comment:
Since we are adding the snippets, we should do it properly and have the actual code not some random `...`